Duplicati/CommandLine/CLI/Strings.cs

```diff
@@ -54,6 +54,12 @@ internal static class Program
         public static string QuietConsoleOptionLong { get { return LC.L(@"If this option is set, progress reports and other messages that would normally go to the console will be redirected to the log."); } }
         public static string QuietConsoleOptionShort { get { return LC.L(@"Disable console output"); } }
         public static string SkippingSourceArgumentsOnNonBackupOperation { get { return @"The --source argument was specified in the parameter file, but the current operation is not a backup operation, so the argument is ignored."; } }
+        public static string AutoUpdateOptionShort { get { return LC.L(@"Toggle automatic updates"); } }
+        public static string AutoUpdateOptionLong { get { return LC.L(@"Set this option if you prefer to have the commandline version automatically update"); } }

```

Comment:
The text of the PR didn't mention this feature; that the CLI also supports auto updating Duplicati.

Reply:
These strings are actually just moved from `Program.cs`. For some reason those two strings were placed in the code file instead of the string file.